internal/pkg/deploy/cloudformation/stack/lb_grpc_web_service_integration_test.go

```diff
@@ -48,11 +47,11 @@ func TestGrpcLoadBalancedWebService_Template(t *testing.T) {
 		require.NoError(t, err)
 		envMft, err := mft.ApplyEnv(tc.envName)
 		require.NoError(t, err)
-
 		err = envMft.Validate()
 		require.NoError(t, err)
+		content := envMft.Manifest()
```

Comment:
If we add it it is not local anymore lol

Reply:
It is not local only if it has `subnet.from_tags` and not `subnet.ids` right?

Reply:
`Load` is supposed to load dynamic contents that requires API calls. If the test itself is supposed to be run locally then it doesn't make sense to me to add `.Load` step 🤔

Reply:
I feel like the only reason why these tests have all these `Validate`, `ApplyEnv` calls is because we didn't have a better way to mimic how `cli` package would have manipulate the manifest, than to just hardcoding-ly repeating the codes in `cli`. Without calling `Load`, we are not effectively "mimicking" it.

Another way that I think about it is that I think these integration tests were meant to test that "given a manifest input, do I eventually get an expected template". Without calling `Load` like `cli` does, we can't be confident with our test result.

Reply:
that's true. IMO we need to
1. remove some of our existing integ tests because we have e2e tests now
2. make current local integ tests to be integ tests because we can't generate expected template without making internet.